src/test/java/org/isf/lab/test/TestLaboratory.java

```diff
@@ -94,7 +91,9 @@ public void check(Laboratory laboratory) {
 			assertThat(laboratory.getPatName()).isEqualTo(patName);
 		}
 		assertThat(laboratory.getDate()).isCloseTo(labDate, within(1, ChronoUnit.SECONDS));
-		assertThat(laboratory.getExamDate()).isEqualTo(examDate);
+		// because CreatedDate is set by the auditable class when the datbase is updated the value will differ from
+		// the static examDate value set above.
+		assertThat(laboratory.getCreatedDate()).isCloseTo(examDate, within(25, ChronoUnit.SECONDS));
```

Comment:
I think auditable fields can be ignored in tests

Reply:
Agreed; good catch.